Packs/FeedTAXII/Integrations/FeedTAXII2/FeedTAXII2.yml

```diff
@@ -165,6 +165,11 @@ configuration:
   - Skip indicators with more than a single observation
   required: false
   type: 15
+- additionalinfo: Choose whether to import the XSOAR custom fields(which overwrites the fields from other sources).
```

Comment:
```suggestion
- additionalinfo: Choose whether to import the XSOAR custom fields. Note: this might overwrite the data pulled from other sources.
```